src/bytes.rs

```diff
@@ -731,20 +731,23 @@ impl From<Vec<u8>> for Bytes {
         let slice = vec.into_boxed_slice();
```

Comment:
The is_empty special case above can be removed now that the low bit is being tracked properly.